tests/wallet_tools.py

```diff
@@ -103,36 +86,7 @@ def make_solution(self, condition_dic: Dict[ConditionOpcode, List[ConditionWithA
 
         for con_list in condition_dic.values():
             for cvp in con_list:
-                if cvp.opcode == ConditionOpcode.CREATE_COIN:
-                    ret.append(make_create_coin_condition(cvp.vars[0], cvp.vars[1]))
-                if cvp.opcode == ConditionOpcode.CREATE_COIN_ANNOUNCEMENT:
-                    ret.append(make_create_coin_announcement(cvp.vars[0]))
-                if cvp.opcode == ConditionOpcode.CREATE_PUZZLE_ANNOUNCEMENT:
-                    ret.append(make_create_puzzle_announcement(cvp.vars[0]))
-                if cvp.opcode == ConditionOpcode.AGG_SIG_UNSAFE:
-                    ret.append(make_assert_aggsig_condition(cvp.vars[0]))
-                if cvp.opcode == ConditionOpcode.ASSERT_COIN_ANNOUNCEMENT:
-                    ret.append(make_assert_coin_announcement(cvp.vars[0]))
-                if cvp.opcode == ConditionOpcode.ASSERT_PUZZLE_ANNOUNCEMENT:
-                    ret.append(make_assert_puzzle_announcement(cvp.vars[0]))
-                if cvp.opcode == ConditionOpcode.ASSERT_SECONDS_ABSOLUTE:
-                    ret.append(make_assert_absolute_seconds_exceeds_condition(cvp.vars[0]))
-                if cvp.opcode == ConditionOpcode.ASSERT_SECONDS_RELATIVE:
-                    ret.append(make_assert_relative_seconds_exceeds_condition(cvp.vars[0]))
-                if cvp.opcode == ConditionOpcode.ASSERT_MY_COIN_ID:
-                    ret.append(make_assert_my_coin_id_condition(cvp.vars[0]))
-                if cvp.opcode == ConditionOpcode.ASSERT_HEIGHT_ABSOLUTE:
-                    ret.append(make_assert_absolute_height_exceeds_condition(cvp.vars[0]))
-                if cvp.opcode == ConditionOpcode.ASSERT_HEIGHT_RELATIVE:
-                    ret.append(make_assert_relative_height_exceeds_condition(cvp.vars[0]))
-                if cvp.opcode == ConditionOpcode.RESERVE_FEE:
-                    ret.append(make_reserve_fee_condition(cvp.vars[0]))
-                if cvp.opcode == ConditionOpcode.ASSERT_MY_PARENT_ID:
-                    ret.append(make_assert_my_parent_id(cvp.vars[0]))
-                if cvp.opcode == ConditionOpcode.ASSERT_MY_PUZZLEHASH:
-                    ret.append(make_assert_my_puzzlehash(cvp.vars[0]))
-                if cvp.opcode == ConditionOpcode.ASSERT_MY_AMOUNT:
-                    ret.append(make_assert_my_amount(cvp.vars[0]))
+                ret.append([cvp.opcode.value] + cvp.vars)
```

Comment:
this test facility need to not do any validation, to allow tests to create invalid conditions

Reply:
note that all of `wallet_tools.py` is test helper functions